### ISSUE TITLE
Repeating <S-K> moves docs window after the first execution

### ISSUE THREAD

**Issue by devth** [liquidz/vim-iced#110]
First time I use `<S-K>` it shows a help window below:

![image](https://user-images.githubusercontent.com/86107/54869080-5f6e4100-4d59-11e9-815b-51c97002b90e.png)

If I run it again, it moves the help window to the rightmost pane in vim which is disruptive and messes up my layout:

![image](https://user-images.githubusercontent.com/86107/54869094-80cf2d00-4d59-11e9-83c4-7ad19962765a.png)

**Comment by liquidz** [comment]
@devth Thanks for your reporting!

Hmm, this is strange behavior.
vim-iced only move window focus, if document buffer is already opened.
https://github.com/liquidz/vim-iced/blob/0.10.2/autoload/iced/buffer.vim#L81
In my environment, it is not reproduced.

Is there a possibility that other plugins are affecting it?

**Comment by devth** [comment]
I'm not sure 😞 
I have [quite a few](https://github.com/devth/dotfiles/blob/522175dde05c8512b15ecd8e977319182a4a479c/config/nvim/init.vim#L25-L133). I'll see if I can narrow it down.

**Comment by liquidz** [comment]
@devth I found! Following `autocmd` is the cause.
https://github.com/devth/dotfiles/blob/522175dde05c8512b15ecd8e977319182a4a479c/config/nvim/init.vim#L252-L253
```
  " Always open help in vert split
  autocmd FileType help wincmd L
```

vim-iced uses `help` file type for document buffer.
https://github.com/liquidz/vim-iced/blob/0.10.2/autoload/iced/nrepl/document.vim#L99

So.. shoud we have a way to specify document buffer position like stdout buffer?
https://liquidz.github.io/vim-iced/vim-iced.html#g:iced#buffer#stdout#mods

**Comment by devth** [comment]
Wow thanks so much for tracking that down!

I'm fine just removing that line for now. The default positioning works for me. I did notice one other inconsistency between the first time I run `K` and the second: 

- the first time I run `K` my cursor stays positioned in the source code window
- the second (or any subsequent) time I run `K` my cursor jumps to to the help doc window

Do you see the same behavior? Would be good if it stayed in the source in all cases.

**Comment by liquidz** [comment]
@devth 

> Do you see the same behavior? Would be good if it stayed in the source in all cases.

Yes, it is the correct behavior.
https://github.com/liquidz/vim-iced/blob/0.10.2/autoload/iced/buffer.vim#L80-L83
But... you are right. I have never benefited from this behavior.
I'll fix it.

**Comment by liquidz** [comment]
Fixed by #115
